runhouse/resources/hardware/cluster.py

```diff
@@ -877,6 +878,11 @@ def restart_server(
                 self._run_cli_commands_on_cluster_helper([command])
                 logger.debug("Saved user config to cluster")
 
+        allowed_log_levels = logging._nameToLevel.keys()
+        if not logs_level or logs_level not in allowed_log_levels:
+            logs_level = logging.getLevelName(logging.root.level)
```

Comment:
in the parent PR we used `DEFAULT_LOG_LEVEL` as the default, I think we should be consistent since these levels will be different (INFO vs WARNING i believe)